README.md

```diff
@@ -22,6 +22,8 @@ $ go get capnproto.org/go/capnp/v3
 
 **NOTE:** You will need to install the [`capnp` tool](https://capnproto.org/capnp-tool.html) in order to compile your Cap'n Proto schemas into Go.  This package has been tested with version `0.8.0` of the `capnp` tool.
 
+Package might not be found due to how Go's module works, in which case you can try setting `GO111MODULE=on` for the command
```

Comment:
The installation procedure is getting kinda complex, so I think we should simplify and make this part of the default installation procedure.

Could you please integrate this as a (non-optional) part of the shell command?  In other words, change the `go get` line above to something like this ought to do the trick:

```bash
$ GO111MODULE=on go get capnproto.org/go/capnp/v3
```

Reply:
Changed